rules_daml/daml.bzl

```diff
@@ -358,6 +360,7 @@ $$DAMLC test --files {files}
                 )
                 for src in srcs
             ]),
+            target = "--target=" + target if (target) else "",
```

Comment:
Wow, I'm surprised we didn't already have this.

Reply:
me too